packages/now-client/src/utils/index.ts

```diff
@@ -85,7 +85,7 @@ export async function getNowIgnore(path: string | string[]): Promise<any> {
     '.wafpicke-*',
     '.lock-wscript',
     '.env',
-    '.env.build',
+    '.env.*',
```

Comment:
I think you need to modified `now-cli` as well which also maintains an ignore list.

Reply:
@styfle It technically shouldn't matter since it's using `now-client` internally now, but I added it just in case. Should be good to go now

Reply:
@rdev If it doesn't matter, than `packages/now-cli/src/util/ignored.ts` should be removed.